### DISCUSSION TITLE
docker compose pull

### DISCUSSION THREAD

**Discussion by crubik** [xtrime-ru/antizapret-vpn-docker#67] [Q&A] [answered]
после обновлений приложений докера, в WG панели пропал пункт Logout. Теперь при переходе Ip:portWGweb попадаю в список клиентов без запроса авторизации в веб
![Screenshot_2](https://github.com/user-attachments/assets/ddb47f9b-a759-464d-941e-016e814bb295)

**Comment by Dimariqe** [comment, marked as answer]
У вас не установлен пароль в окружении docker-compose.

Перенесите его из файла `./wireguard/wireguard.env` Ниже этой строки:
https://github.com/xtrime-ru/antizapret-vpn-docker/blob/master/docker-compose.wireguard.yml#L21

Или внесите изменения из этого коммита:
https://github.com/xtrime-ru/antizapret-vpn-docker/pull/66/commits/0b65b697811612533c6ce652d66271b151f2f819

UPDATE:
Моя ошибка, не увидел как устанавливался пароль в основной инструкции.

**Reply by xtrime-ru** [reply]
Я тоже не заметил, что из docker-compose.wireguard.yml пропало монтирование env файла. Запушил фикс.
